app/src/main/java/com/example/qrscore/Score.java

```diff
@@ -0,0 +1,5 @@
+package com.example.qrscore;
```

Comment:
For Score.java, we should probably name it QRScore.java as there are scores for both players and QR codes